src/entities/vaccine-registration.entity.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { User } from './user.entity';
+import { VaccineRegistrationResult } from './vaccine-registration-result.entity';
+
+export enum STATUS {
+  REQUESTED = 'REQUESTED',
+  ACCEPTED = 'ACCEPTED',
+  REJECTED = 'REJECTED',
+  COMPLETED = 'COMPLETED',
+}
```

Comment:
Viết theo convention TableNameField
```suggestion
export enum VaccineRegistrationStatus {
  Requested = 'requested',
}
```